src/scripts/devhub.zig

```diff
@@ -129,6 +129,28 @@ fn devhub_metrics(shell: *Shell, cli_args: CLIArgs) !void {
         "us",
     );
 
+    const ci_pipeline_duration_s = blk: {
+        const times_gh = try shell.exec_stdout("gh run list -c {sha} -e merge_group " ++
+            "--json startedAt,updatedAt -L 1 --template {template}", .{
+            .sha = cli_args.sha,
+            .template = "{{range .}}{{.startedAt}} {{.updatedAt}}{{end}}",
+        });
+        var times_iterator = std.mem.split(u8, times_gh, " ");
+        const iso8601_started_at = times_iterator.next() orelse break :blk null;
+        const iso8601_updated_at = times_iterator.next() orelse break :blk null;
+
+        const epoch_started_at = try std.fmt.parseInt(u64, try shell.exec_stdout(
+            "date -d {iso8601_started_at} +%s",
+            .{ .iso8601_started_at = iso8601_started_at },
+        ), 10);
+        const epoch_updated_at = try std.fmt.parseInt(u64, try shell.exec_stdout(
+            "date -d {iso8601_updated_at} +%s",
+            .{ .iso8601_updated_at = iso8601_updated_at },
+        ), 10);
```

Comment:
I'll admit this code is a bit suspect, calling `date` to do ISO8601 to epoch conversion, but I didn't particular fancy writing a parser for it then dealing with `epoch.zig` :sweat_smile:.

Reply:
SGTM, but let's extract this into:

```zig
fn iso8601_to_timestamp_seconds(shell: *Shell, datetime: []const u8) !u64
```

so that it would be easier to write a parser in the future (I bet a sufficiently smart LLM can do it)